docs/plugin-transform-typescript.md

```diff
@@ -153,19 +153,27 @@ If you have existing code which uses the TypeScript-only [namespace][namespace]
 
 `boolean`, defaults to `false`
 
+Forcibly enables `jsx` parsing. Otherwise angle brackets will be treated as typescript's legacy type assertion `var foo = <string>bar;`. Also, `isTSX: true` requires `allExtensions: true`.
```

Comment:
Nit:

```suggestion
Forcibly enables `jsx` parsing. Otherwise angle brackets will be treated as TypeScript's legacy type assertion `var foo = <string>bar;`. Also, `isTSX: true` requires `allExtensions: true`.
```